Mathlib/Data/Vector/Basic.lean

```diff
@@ -522,16 +522,14 @@ def casesOn {motive : ∀ {n}, Vector α n → Sort*} (v : Vector α m)
 
 /-- Define `motive v₁ v₂` by case-analysis on `v₁ : Vector α n` and `v₂ : Vector β n` -/
 def casesOn₂  {motive : ∀{n}, Vector α n → Vector β n → Sort*} (v₁ : Vector α m) (v₂ : Vector β m)
-              (nil : motive nil nil)
-              (cons : ∀{n}, (x : α) → (y : β) → (xs : Vector α n) → (ys : Vector β n)
-                      → motive (x ::ᵥ xs) (y ::ᵥ ys)) :
-              motive v₁ v₂ :=
-    inductionOn₂ (C := motive) v₁ v₂ nil @fun _ x y xs ys _ => cons x y xs ys
+    (nil : motive nil nil) (cons : ∀{n}, (x : α) → (y : β) → (xs : Vector α n) → (ys : Vector β n)
+    → motive (x ::ᵥ xs) (y ::ᵥ ys)) : motive v₁ v₂ :=
+  inductionOn₂ (C := motive) v₁ v₂ nil @fun _ x y xs ys _ => cons x y xs ys
```

Comment:
I don't think this is an improvement. Moving things back to the normal indentation is fine, but having each argument on its own line with extra indentations makes things clearer here imho.

Reply:
moved back to old line, but with normal indentation.

Reply:
Should be fixed now